packages/next-swc/crates/next-transform-dynamic/src/lib.rs

```diff
@@ -665,3 +649,25 @@ fn rel_filename(base: Option<&Path>, file: &FileName) -> String {
 
     rel_path.display().to_string()
 }
+
+fn with_chunking_type(chunking_type: &str) -> Box<ObjectLit> {
+    with_clause(&[("chunking-type", chunking_type)])
```

Comment:
should we prefix these with `turbopack` to avoid potential conficts and confusion?


```suggestion
    with_clause(&[("turbopack-chunking-type", chunking_type)])
```